tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -112,60 +86,17 @@ class WskBasicUsageTests extends TestHelpers with WskTestHelpers {
         rule.create(ruleName, trigger = triggerName, action = fullQualifiedName)
       }
 
-      wsk.action.invoke(fullQualifiedName).stdout should include(s"ok: invoked /$fullQualifiedName")
-      wsk.action.get(fullQualifiedName).stdout should include(s"ok: got action ${packageName}/${actionName}")
-  }
-
-  it should "include CLI user agent headers with outbound requests" in {
-    val stdout = wsk.cli(Seq("list", "--auth", wskprops.authKey) ++ wskprops.overrides, verbose = true).stdout
-    stdout should include regex (usrAgentHeaderRegEx)
+      val run = wsk.action.invoke(fullQualifiedName)
+      withActivation(wsk.activation, run) { activation =>
+        activation.response.status shouldBe "success"
+      }
+      val action = wsk.action.get(fullQualifiedName)
+      action.getField("name") shouldBe actionName
+      action.getField("namespace") shouldBe s"${guestNamespace}/${packageName}"
   }
 
   behavior of "Wsk actions"
 
-  it should "reject creating entities with invalid names" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
I think this test is applicable with wskRest

Reply:
done.